rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1057,6 +1058,18 @@ public long getLockingCap(Object[] args) {
         return lockingCap.getValue();
     }
 
+    public byte[] getActivePowpegRedeemScript(Object[] args) {
```

Comment:
We are missing tests for this method. Check `BridgeTest` class

Reply:
What if I remove the test I just created, and create new tests just for this method? As this method acts as entrypoint for the other one, I would be testing both at once. What do you think?